docs/content/documentation/getting-started/overview.md

```diff
@@ -219,7 +219,7 @@ The `index.html` file inside the `templates` directory should be:
 <h1 class="title">
   This is my blog made with Zola.
 </h1>
-<p>Click <a href="/blog/">here</a> to see my posts.</p>
+<p>Click <a href="{{ get_url(path="@/blog/_index.md") }}">here</a> to see my posts.</p>
```

Comment:
My bad it should be `{{/* get_url(path="@/blog/_index.md") */}}` to avoid zola trying to load it

Reply:
Ah good catch, thanks! I've changed this.

 While serving the docs locally I noticed 4 broken anchors. 

I think that the anchors are broken because the headings have a `/` and this  character is not stripped for the slugs. Should this be added to the list of characters? I can open a new issue and PR for this. I'm really liking Zola and I'm looking for ways to help out.
```
> Checked 79 internal link(s) with anchors: 4 target(s) missing.
Error: The anchor in the link `@/themes/papaya/index.md#postproject-date-formats` in /Users/danielrb/open/zola/docs/content/themes/papaya/index.md does not exist.
The anchor in the link `@/themes/papaya/index.md#postproject-featured-images` in /Users/danielrb/open/zola/docs/content/themes/papaya/index.md does not exist.
The anchor in the link `@/themes/papaya/index.md#open-graph-protocol-localeprofile-information` in /Users/danielrb/open/zola/docs/content/themes/papaya/index.md does not exist.
The anchor in the link `@/themes/papaya/index.md#socialcontact-links` in /Users/danielrb/open/zola/docs/content/themes/papaya/index.md does not exist.
```

Reply:
We should ignore errors in themes directory since it's essentially coming from GitHub READMEs and their slugification differs from Zola so we might always have broken intra-links.
I can't remember if that's possible yet or not though.